### PR TITLE
[WIP]collection of Config management

### PR DESCRIPTION
**Pull request by reubenvarghese1**
Purpose or Intent
=================

* Collection of Config Management

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 6dfd16e2685e3bb4afb8d7114d7b3a99d72b3fcb:

`cfme/infrastructure/config_management.py`:
- :bangbang: Line 508:1: **E302** *expected 2 blank lines, found 1*
- :bangbang: Line 532:101: **E501** *line too long (119 > 100 characters)*
- :bangbang: Line 585:101: **E501** *line too long (126 > 100 characters)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: entry_points.txt [hunk 1]
```diff
@@ -39,6 +40,9 @@ cloud_providers = cfme.cloud.provider:CloudProviderCollection
 cloud_stacks = cfme.cloud.stack:StackCollection
 cloud_tenants = cfme.cloud.tenant:TenantCollection
 clusters = cfme.infrastructure.cluster:ClusterCollection
+config_manager = cfme.infrastructure.config_management:ConfigManagersCollection
+config_profile = cfme.infrastructure.config_management:ConfigProfilesCollection
+config_system = cfme.infrastructure.config_management:ConfigSystemsCollection
```
Comment (sshveta) [inline entry_points.txt]: All collections need to be plural like the above lines 40, 41, 42
config_managers = cfme.infrastructure.config_management:ConfigManagersCollection
config_profiles = cfme.infrastructure.config_management:ConfigProfilesCollection
config_systems = cfme.infrastructure.config_management:ConfigSystemsCollection

File: entry_points.txt [hunk 2]
```diff
@@ -14,6 +14,7 @@ analysis_profiles = cfme.configure.configuration.analysis_profile:AnalysisProfil
 ansible_credentials = cfme.ansible.credentials:CredentialsCollection
 ansible_playbooks = cfme.ansible.playbooks:PlaybooksCollection
 ansible_repositories = cfme.ansible.repositories:RepositoryCollection
+ansible_tower = cfme.infrastructure.config_management:AnsibleTowersCollection
```
Comment (sshveta) [inline entry_points.txt]: This needs to be plural.
May be :
config_manager_ansible_tower_providers = cfme.infrastructure.config_management:AnsibleTowersCollection

File: entry_points.txt [hunk 3]
```diff
@@ -94,6 +98,7 @@ requests = cfme.services.requests:RequestCollection
 resource_pools = cfme.infrastructure.resource_pool:ResourcePoolCollection
 roles = cfme.configure.access_control:RoleCollection
 saved_reports = cfme.intelligence.reports.saved:SavedReportsCollection
+satellite = cfme.infrastructure.config_management:SatellitesCollection
```
Comment (sshveta) [inline entry_points.txt]: Same here .
satellites = cfme.infrastructure.config_management:SatellitesCollection
or 
satellite_providers = cfme.infrastructure.config_management:SatellitesCollection

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by mshriver** [comment]
Superseded by @john-dupuy work, and ConfigManager will be defined as an actual provider class.

**Pull request closed without merging**